src/main/java/com/minecolonies/coremod/sounds/DeliverymanSounds.java

```diff
@@ -0,0 +1,147 @@
+     * Chance to say a phrase.
+     */
+    private static final int    PHRASE_CHANCE      = 25;
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int    BASIC_SOUND_CHANCE = 100;
+    /**
+     * Random generator.
+     */
+    private static final Random rand               = new Random();
+
+    /**
+     * Containing the female fisherman sounds.
+     */
+    public static final class Female
+    {
+        public static final SoundEvent generalPhrases = ModSoundEvents.getSoundID("mob.deliveryman.female.generalPhrases");
+        public static final SoundEvent noises         = ModSoundEvents.getSoundID("mob.deliveryman.female.noise");
+        public static final SoundEvent hostile        = ModSoundEvents.getSoundID("mob.deliveryman.female.hostile");
+        public static final SoundEvent offToBed       = ModSoundEvents.getSoundID("mob.deliveryman.female.offToBed");
+        public static final SoundEvent badWeather     = ModSoundEvents.getSoundID("mob.deliveryman.female.badWeather");
+        public static final SoundEvent saturationVeryLow     = ModSoundEvents.getSoundID("mob.deliveryman.female.saturationVeryLow");
+        public static final SoundEvent saturationLow     = ModSoundEvents.getSoundID("mob.deliveryman.female.saturationLow");
+        public static final SoundEvent saturationHigh     = ModSoundEvents.getSoundID("mob.deliveryman.female.saturationHigh");
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Female()
+        {
+            /*
+              Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * Containing the male fisherman sounds.
+     */
+    public static final class Male
+    {
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Male()
+        {
+            /*
+             * Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private DeliverymanSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+
+    /**
+     * Plays fisherman sounds.
+     * We ignore the warning since the case integers counts for the amount of different sound events and we don't have to name them.
+     * @param worldIn  the world to play the sound in.
+     * @param position the position to play the sound at.
+     * @param isFemale the gender.
+     * @param saturation the saturation.
+     */
+    @SuppressWarnings("squid:S109")
+    public static void playDmanSounds(final World worldIn, final BlockPos position, final boolean isFemale, final double saturation)
+    {
+        //While there are no male sounds
+        if(!isFemale)
+        {
+            return;
+        }
+        //Leaving it as switch-case we may add further random sound categories here (Whistling, singing, etc).
+        switch (rand.nextInt(NUMBER_OF_SOUNDS + 1))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)